### PR TITLE
Fixed QR issue

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
Fixed issue in the link creation

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
@mustafapylons @heyitsjk  can you look into the translations please.

File: easel/i18n/de.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "upload_to_ipfs": "Auf IPFS hochladen",
   "content_id": "Inhalts-ID",
   "tx_receipt": "TX-Empfang",
-  "close": "Fenster schließen",
+  "close": "Nah dran",
```
Comment (mustafapylons) [inline easel/i18n/de.json:19]: correct translation should be "Anwendungsfenster schliessen"

**Review by mustafapylons — COMMENTED** [review]
One small translation issue, will review more.

**Comment by kjawadDeveloper2** [comment]
Resolves this: https://github.com/Pylons-tech/pylons/issues/1048

**Review by faisalnaveedRNS — APPROVED** [review]

File: easel/i18n/de.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "upload_to_ipfs": "Auf IPFS hochladen",
   "content_id": "Inhalts-ID",
   "tx_receipt": "TX-Empfang",
-  "close": "Fenster schließen",
+  "close": "Schließen",
```
Comment (mustafapylons) [inline easel/i18n/de.json:19]: This is approved to shorten the string for the screen, it's not full translation.

File: wallet/pkgs/pdf_viewer-master/android/src/main/java/pt/tribeiro/flutter_plugin_pdf_viewer/FlutterPluginPdfViewerPlugin.java [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+        try {
+            File directory = context.getCacheDir();
+            FilenameFilter myFilter = new FilenameFilter() {
+                @Override
+                public boolean accept(File dir, String name) {
+                    return name.toLowerCase().startsWith(filePrefix.toLowerCase());
+                }
+            };
+            File[] files = directory.listFiles(myFilter);
+            // Log.d("Cache Files", "Size: " + files.length);
```
Comment (mustafapylons) [inline wallet/pkgs/pdf_viewer-master/android/src/main/java/pt/tribeiro/flutter_plugin_pdf_viewer/FlutterPluginPdfViewerPlugin.java:137]: Let's remove this dead code.

**Review by mustafapylons — COMMENTED** [review]
Few small comments, otherwise code is good.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Pull request merged**